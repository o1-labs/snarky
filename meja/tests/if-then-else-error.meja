let x = fun (b) => {
  if (b) {
    true;
  } else {
    1;
  };
};
