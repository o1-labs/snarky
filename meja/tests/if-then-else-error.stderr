File "tests/if-then-else-error.meja", line 5, characters 4-5:
Error: Incompatible types bool and int:
       Cannot unify bool and int
